src/adagrams.js

```diff
@@ -1,8 +1,97 @@
 const Adagrams = {
   drawLetters() {
     // Implement this method for wave 1
+    const letterPool = ( "A" * 9 + "B" * 2 + "C" * 2 + "D" * 4 + "E" * 12 + "F" * 2 + "G" * 3 +
```

Comment:
This data structure works, but it's not very DRY and would be tricky to change. For example, suppose I said you had the wrong number of "I"s - you would have to do a lot of counting to solve the problem.

Instead you might store a hash of letter frequencies like this

```
letterQuantities: {
  A: 9, B: 2, ...
}
```
Also, best practice is to move this object outside of the `drawLetters()` function, but inside the `Adagrams` module.